services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket],
+                 username: str,
+                 date: str = None) -> QuerySet:
+    user = User.objects.get(username=username)
```

Comment:
Here you are using `User` directly instead of `get_user_model()`. Replace `User` with `get_user_model()`. This is a best practice for Django.